src/orquestra/sdk/_base/_in_process_runtime.py

```diff
@@ -224,3 +261,13 @@ def list_workflow_runs(
                 -limit:
             ]
         return wf_runs
+
+    @classmethod
+    def from_runtime_configuration(cls, *args, **kwargs):
+        raise NotImplementedError()
+
+    def get_all_workflow_runs_status(self, *args, **kwargs):
```

Comment:
suggestion: maybe better exception? Meaning like "method {} is not implemented for in-process runtime". WDYT

Reply:
Added :)